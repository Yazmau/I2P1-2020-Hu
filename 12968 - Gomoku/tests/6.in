30
....b.......b..
b..b.....b....w
...........b...
...............
.w....b.......b
...............
......w..bb....
...............
.w.............
........w......
..............w
....w..........
...........w...
......w........
......w........
...............
...............
.........b.....
...............
...............
..bbb..........
..bwb..........
..bwb..........
...w...........
...w...........
...w...........
.b.b...........
...........w...
...............
..w.........www
...............
.............b.
.....b.........
....bbbw.......
....bwwwwwb....
....bbb........
...............
...............
.............w.
...............
...............
...............
...............
...............
............www
.......www.....
b......wbw.....
.......bww.....
......b.....b..
.....b.........
....b..........
...w...........
...............
...............
...w...........
..............w
...............
...............
...............
...........bbbb
...............
b..............
.b.........w..b
b............w.
.............b.
............b..
......b....b...
.....b......w..
...............
...............
....wb.........
......www..w...
w..............
...............
..............w
........www....
........wbw....
........wbw...b
.........b..b..
.........b.....
.........b.....
.........w.....
...............
......w........
...............
...............
...............
...............
...............
w..........bbbb
...............
..........b....
...............
...............
...............
.b.............
...............
........www....
........wbbbbbw
........www....
...............
........w......
...............
...w...........
...........bbbb
...............
...............
...............
..............b
...............
...............
...............
......b..bbb...
.........bwb...
.........wbb...
........w......
.......w.......
......w...w....
.....b.........
.........w..www
....b......b...
...............
.............b.
...............
b......bw......
.......w.....w.
.b............b
........b.....b
...............
bw...w.........
.........w.....
..........w....
w...w..........
...............
.......w......b
...............
...............
...............
......b........
...............
......bbb......
.....wbwb......
......bwb......
.......w.......
..b....w.......
..w....w.......
.......b.......
...............
...............
............www
............b..
...............
...............
.b.............
...............
...............
...............
...............
....www........
.ww.wbbbbbw....
....www........
...............
...............
...............
...........bbbb
...............
..........www..
..........wbw..
...b......bww..
.........b.....
.w......b......
.......b.......
......w........
...............
...............
...............
...........b...
...............
...............
....w......bbbb
..............b
...............
.............b.
............w..
.........b.....
....b...b...b..
.......w..b....
...............
.........b.....
.............w.
..........w....
..........w....
..........bb...
w.....w......ww
.....w........b
...............
.b.............
...............
...............
...............
...............
...............
.......www.....
.......wbw.....
.......wbw.....
....b...bw.w...
........b......
........b......
........w......
...........bbbb
...............
...............
...............
.........b.....
...............
...............
..............b
...............
...............
.............w.
...............
...............
..bbb..........
..bwwwwwb......
..bbb.....w.www
...............
...............
...............
...............
..............b
............www
............wbw
............bww
...........b...
..........b....
........wb.....
........w......
...........b...
..........w....
...........bbbb
..b............
...............
.........b.....
......b.w..w...
........b....b.
............w..
...b.b..w....w.
...............
...b.w.........
...............
......b.w......
...............
...............
...........b...
w.w..........w.
...............
...........b...
...............
...............
......bbb.b....
......bwb......
......bwb......
.......w.......
.......w.......
.......w.......
.......b.......
.........w.....
...............
.............w.
............www
...............
.......b.......
...............
...............
...............
..........b....
......bbb......
......bwwwwwb..
......bbb......
...............
.w.............
...............
....w..........
...............
............www
...............
...............
...............
...........b...
...............
...b...........
w..............
...............
.......bbb.....
w......bwb.....
.......wbb.....
......w........
.....w.........
....w..........
...b........www
...............
.b.....b.......
...............
.b...b.b.......
...............
..............b
.b.............
...............
..........w.b..
........w.b....
....b..........
.ww.....w.w.w..
...............
..............w
.........w...w.
...............
........b......
........w......
.......b.......
...............
...........www.
...........wbw.
...........wbw.
............b..
............b..
............b..
............w..
...............
...............
......w....bbbb
...............
........b......
...............
...............
.......b.......
.......bbb.....
.......bwwwwwb.
.......bbb.....
..........w....
...............
...............
....w..........
...............
...............
............www
...............
...............
...............
...............
...............
...............
...............
.........b.....
.....www.......
.....wbw.w.....
.....bww.......
....b..........
...b..b........
..b...w........
.w.........bbbb
.......b....w..
b.....b...b....
....b..........
...............
...............
...b.....w.....
..b.....w......
...............
.b.......w.b...
..........wb...
...............
....w..........
...w...........
.w......w......
..............w
............bbb
............bwb
...........bbwb
.............w.
.............w.
.............w.
.............b.
........b......
....w..........
...............
........w......
...............
...............
...............
............www
...............
...............
...............
...............
......www......
.....bwbbbbbw..
..w...www......
...............
..............b
...............
...............
...............
...............
...............
....w......bbbb
...............
......www......
......wbw.b....
......bww......
.....b.........
....b..........
...b...........
..w.....b......
..........w....
.w.............
...............
...............
...............
...............
...........bbbb
.........bb...b
.b.............
........b......
...w.b..w......
...............
..........b....
...........b...
b..............
........w......
w.......w.....b
..........w....
....w.....w....
............w..
..........w....
..............b
...............
...............
...............
...............
....b..........
....www........
....wbw........
....wbw........
.....b..w......
....bb.........
.....b.........
....ww.........
...............
...............
...........bbbb
