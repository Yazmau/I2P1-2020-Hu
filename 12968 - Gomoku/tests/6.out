none
white
white
black
none
black
black
white
none
white
black
black
none
black
white
black
none
white
white
white
none
black
white
black
none
white
black
black
none
black
